firmware/thruna/BMS/src/app/app_imd.c

```diff
@@ -7,7 +7,9 @@
 #include "app_canTx.h"
 #include "app_utils.h"
 
+// PWM_TICKS_MAX also defined and explained in "hw_pwmInputs.c"
 #define IMD_FREQUENCY_TOLERANCE 2.0f
+#define PWM_TICKS_MAX 255
```

Comment:
Maybe insteadjust use #define MAX_8_BITS_VALUE (uint32_t)(255) which already exists in app_utils.h?